Doxyfile

```diff
@@ -0,0 +1,116 @@
+# Doxyfile 1.9.4
+
+# This file describes the settings to be used by the documentation system
+# doxygen (www.doxygen.org) for a project.
+#
+# All text after a double hash (##) is considered a comment and is placed in
+# front of the TAG it is preceding.
+#
+# All text after a single hash (#) is considered a comment and will be ignored.
+# The format is:
+# TAG = value [value, ...]
+# For lists, items can also be appended using:
+# TAG += value [value, ...]
+# Values that contain spaces should be placed between quotes (\" \").
+#
+# Note:
+#
+# Use doxygen to compare the used configuration file with the template
+# configuration file:
+# doxygen -x [configFile]
+# Use doxygen to compare the used configuration file with the template
+# configuration file without replacing the environment variables:
+# doxygen -x_noenv [configFile]
+
+DOXYFILE_ENCODING      = UTF-8
+PROJECT_NAME           = "Endless Sky"
+PROJECT_NUMBER         =
+PROJECT_BRIEF          = "This is an auto generated documentation for the source code of endless-sky."
+PROJECT_LOGO           = icons/icon_48x48.png
+OUTPUT_DIRECTORY       = ./docs/api
+
+CREATE_SUBDIRS         = YES
+
+INPUT                  = ./source
```

Comment:
is this including the subdirs? iirc I had to add them manually

Reply:
Yes, that is handled by the `RECURSIVE = YES` option just below it.

Reply:
ah ok :+1: